objcgen/objcgenerator-postprocessor.cs

```diff
@@ -60,10 +69,25 @@ protected IEnumerable<ProcessedProperty> PostProcessProperties (IEnumerable<Prop
 		{
 			foreach (PropertyInfo property in properties) {
 				ProcessedProperty processedProperty = new ProcessedProperty (property, this);
+
+				if (!ProcessPotentialName (processedProperty))
+					continue;
+
 				yield return processedProperty;
 			}
 		}
 
+		bool ProcessPotentialName (ProcessedProperty processedProperty)
+		{
+			switch (processedProperty.Name) {
+				case "setClass":
+				case "class":
+					Delayed.Add (ErrorHelper.CreateWarning (1051, $"Element '{processedProperty.Property.Name}' is not generated because its name conflicts with important objective-c selector '{processedProperty.Name}'"));
+					return false;
```

Comment:
I think you can add all the selectors in `NSObjectProtocol`: https://developer.apple.com/reference/objectivec/1418956-nsobject?language=objc